scripts/linux/publishReleasePackages.sh

```diff
@@ -3,9 +3,10 @@
 #Pre-Requisites For Running Locally
 #Docker Connection to MSINT
 #AZ CLI LOGIN
-
+SCRIPT_NAME=$(basename $0)
```

Comment:
Variable not used?

Reply:
It is used on line 14

Reply:
Duh...that line wasn't loaded into the diff. 🙄Thanks!